### ISSUE TITLE
ARM iso

### ISSUE THREAD

**Issue by Tangerinrs** [Athena-OS/athena-iso#15]
Can you provide the iso of the arm? Thank you

**Comment by D3vil0per** [comment]
> Can you provide the iso of the arm? Thank you

Hey @Tangerinrs Currently I don't have time effort and a ARM-based machine for testing an ISO. If there is one would like to collaborate on this path, he/she is welcome

**Comment by AbhinavRathod** [comment]
- Thanks for athena os and hope someone will try to help for creating arm iso also Your athena os is really great for pentesting I loved it but I would like it to be running on MacBook natively because after asahi linux fedora also created a same way to boot Linux natively on mac and it will be great full to see that running on mac plus it will help for easier pentesting btw it's just an idea 
Beside that it will be good to get an arm iso cuz pentesting box can also be virtualised on vm

**Comment by D3vil0p3r** [comment]
Dear @AbhinavRathod I remember months ago a user in our Discord channel that was able to run Athena in a Apple M1 (that should be ARM-based) but I don't remember how he did... For sure I want to implement it because I have a good number of users asking me this. I have a Intel processor so, if I should create an ARM ISO and I want to test it, I don't know if there are some software that can emulate ARM processors where I can test this ARM ISO.

**Comment by 8b17-r48817** [comment]
Qemu should work to emulate ARM

**Comment by AkechiShiro** [comment]
@8b17-r48817 I don't think it's a good idea to use a VM inside an ISO to install on a target machine, Athena Nyx can be used on ARM but installation steps might be advanced for users non familiar using Nix.

**Comment by AkechiShiro** [comment]
We're also thinking of plans for RISC-V support

**Comment by AkechiShiro** [comment]
For ArchLinux ARM, there is this : https://archlinuxarm.org/

**Comment by 0xErrors** [comment]
> @8b17-r48817 I don't think it's a good idea to use a VM inside an ISO to install on a target machine, Athena Nyx can be used on ARM but installation steps might be advanced for users non familiar using Nix.

yeah but it needs some pkg to configure for arm cuz building athena-nix on arm took too much time for me so i gave up 😅 if you try and was able to make athena-nix-arm then ping me in athena discord i would like to test it 🙂
